games-simulation/simutrans/simutrans-nightly.recipe

```diff
@@ -9,18 +9,17 @@ changes which make your savegames unreadable by any official version, or \
 bugs that will destroy your game, or not even run. They are for bug testing \
 only."
 HOMEPAGE="http://www.simutrans.com"
-SOURCE_URI="svn://tron.yamagi.org/simutrans/simutrans/trunk \
-	--username=anon --password="
+SOURCE_URI="svn://servers.simutrans.org/simutrans/trunk"
 SOURCE_FILENAME="simutrans-$portVersion"
 SOURCE_DIR="trunk"
 COPYRIGHT="1997-2004 Hj. Malthaner
-	2005-2015 The Simutrans Team"
+	2005-2017 The Simutrans Team"
 LICENSE="Artistic"
-REVISION="2"
+REVISION="3"
 ARCHITECTURES="!x86_gcc2 ?x86 x86_64"
 SECONDARY_ARCHITECTURES="x86"
 PROVIDES="
-	simutrans$secondaryArchSuffix = $portVersion
+	simutrans$secondaryArchSuffix = $portVersion compat >= 120.2
```

Comment:
Okay, just for clarification, haikuporter would "count" like this:
1.2.1~nightly < 1.2.1 ?
1.2 > 1.2.0 ? (I had to add compat >= 120.2 to pakset 120.2.0 to get it past the check of "simutrans_pakset$secondaryArchSuffix >= 120.2".
BTW, now that the paksets (apart from the nightly ones) are back to "any" architecture, is it still okay to use "$secondaryArchSuffix" here? On 64bit you can still combine nightly simutrans package with release pakset or the other way round, but what happens on gcc2hybrid?